lib/ftfunctions/java.vim

```diff
@@ -0,0 +1,23 @@
+vim9script
+
+#=====
+# this function  parse caller through 
+# reg exp and populate outline window
+# ====
+
+import autoload "../regex.vim"
+
+export def CurrentItem(curr_item: string): string
+    #return trim(matchstr(curr_item, '(.*'))
+    return curr_item
+enddef
+
+export def FilterOutline(outline: list<string>): list<string>
+    outline ->filter("v:val =~ " .. string(join(regex.outline_pattern_to_include["java"], '\|')))
+    outline ->filter("v:val =~ " .. string(join(regex.outline_pattern_to_exclude["java"], '\|')))
```

Comment:
```diff
-    outline ->filter("v:val =~ " .. string(join(regex.outline_pattern_to_exclude["java"], '\|')))
+            ->filter("v:val =~ " .. string(join(regex.outline_pattern_to_exclude["java"], '\|')))
```

Reply:
You can chain filters.

Reply:
applied the suggestion in latest commit, please review